acme/acme/__init__.py

```diff
@@ -10,3 +10,12 @@
   https://github.com/ietf-wg-acme/acme/tree/draft-ietf-acme-acme-01
 
 """
+import sys
+import warnings
+
+if sys.version_info[:2] == (3, 3):
+    warnings.warn(
+            "Python 3.3 support will be dropped in the next release of "
+            "Certbot and acme. Please upgrade your Python version.",
```

Comment:
I'm sorry that my previous comments were ambiguous. While I'm glad you included acme, I don't think we should mention Certbot at all here. [Many](https://github.com/zenhack/simp_le) [other](https://github.com/costela/wile) [unrelated](https://github.com/adfinis-sygroup/acme-powerdns/blob/master/setup.py) [projects](https://github.com/alex/letsencrypt-aws) use our ACME library and Certbot users will get the warning from Certbot itself.